jobs/JGLOBAL_ATMENS_ANALYSIS_INITIALIZE

```diff
@@ -7,19 +7,33 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "atmensanlinit" -c "base atmensanl atm
 ##############################################
 # Set variables used in the script
 ##############################################
+# shellcheck disable=SC2153
 GDATE=$(date +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
+gPDY=${GDATE:0:8}
+gcyc=${GDATE:8:2}
 GDUMP="gdas"
+GDUMP_ENS="enkf${GDUMP}"
 
 
 ##############################################
 # Begin JOB SPECIFIC work
 ##############################################
-export COMOUT=${COMOUT:-${ROTDIR}/${RUN}.${PDY}/${cyc}}
-mkdir -p "${COMOUT}"
 
-# COMIN_GES and COMIN_GES_ENS are used in script
-export COMIN_GES="${ROTDIR}/${GDUMP}.${GDATE:0:8}/${GDATE:8:2}/atmos"
-export COMIN_GES_ENS="${ROTDIR}/enkf${GDUMP}.${GDATE:0:8}/${GDATE:8:2}"
+# Generate COM variables from templates
+RUN=${GDUMP} YMD=${PDY} HH=${cyc} generate_com -rx COM_OBS COM_ATMOS_ANALYSIS
+
+RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx \
+    COM_ATMOS_ANALYSIS_PREV:COM_ATMOS_ANALYSIS_TMPL 
+
+RUN=${GDUMP_ENS} YMD=${PDY} HH=${cyc} generate_com -rx \
+    COM_TOP_ENS:COM_TOP_TMPL
+
+RUN=${GDUMP_ENS} YMD=${gPDY} HH=${gcyc} generate_com -rx \
+    COM_TOP_ENS_GES:COM_TOP_TMPL
+
+
+mkdir -m 775 -p "${COM_TOP_ENS}"
```

Comment:
```suggestion
```
Can add the mkdir in the python where you generate the path.

Reply:
Requested change committed at [`dc0f196`](https://github.com/RussTreadon-NOAA/global-workflow/commit/dc0f1963c9088ed946661c29e196c6fba42ecf5c)